package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "angular-bootstrap",
+    "version": "0.12.0",
+    "description": "Native AngularJS (Angular) directives for Bootstrap.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/angular-ui/bootstrap-bower"
+    },
+    "author": {
+        "name": "https://github.com/angular-ui/bootstrap/graphs/contributors"
+    },
+    "keywords": [
+        "angular",
+        "angular-ui",
+        "bootstrap"
+    ],
+    "license": "MIT",
+    "ignore": [],
+    "main": ["./ui-bootstrap-tpls.js"],
```

Comment:
this one breaks browserify -- it should be one file, not an array: `"main": "./ui-bootstrap-tpls.js",`